src/dense.jl

```diff
@@ -323,26 +347,10 @@ mode(::AutoSymbolics) = SymbolicMode()
 """
     AutoTapir
 
-Struct used to select the [Tapir.jl](https://github.com/withbayes/Tapir.jl) backend for automatic differentiation.
-
-Defined by [ADTypes.jl](https://github.com/SciML/ADTypes.jl).
-
-# Constructors
-
-    AutoTapir(; safe_mode=true)
-
-# Fields
-
-  - `safe_mode::Bool`: whether to run additional checks to catch errors early. While this is
```

Comment:
Maybe we could preserve this part of the old docstring, so that future maintainers of ADTypes know it should just be left that way (rather than deleted)?

Reply:
But we put the big red warning right on top